nat-lab/tests/telio.py

```diff
@@ -177,6 +184,22 @@ class AdapterType(Enum):
     WireguardGo = "wireguard-go"
     WindowsNativeWg = "wireguard-nt"
 
+    def convert_adapter_type(self, router: Router) -> libtelio.TelioAdapterType:
+        match self:
```

Comment:
are we ready to break backwards compatibility with python3.9? just curious, not against it :laughing:

Reply:
I just jumped at the suggestion without checking compat :joy: But I agree, let's not break that compatibility quite yet